eng/AfterSigning.targets

```diff
@@ -8,4 +8,24 @@
       Text="$(RazorExtensionVSIXName) was not generated."
       Condition="!Exists('$(VSSetupDir)$(Configuration)\$(RazorExtensionVSIXName)')" />
   </Target>
+
+  <Target Name="_ZipLanguageServerBinaries" AfterTargets="Pack"  Condition=" '$(OS)' == 'Windows_NT' ">
+    <!-- Build a .zip file from each platform's directory and write it to 'artifacts' -->
+    <PropertyGroup>
+      <RidsPublishDir>$(ArtifactsDir)LanguageServer\$(Configuration)\</RidsPublishDir>
+      <ZipOutputDir>$(RidsPublishDir)</ZipOutputDir>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <LanguageServiceBinaryDir Include="$([System.IO.Directory]::GetDirectories(&quot;$(RidsPublishDir)&quot;))" />
+      <LanguageServiceBinary Include="@(LanguageServiceBinaryDir)">
+        <SourceDir>%(LanguageServiceBinaryDir.Identity)</SourceDir>
+        <ZipFile>$(ZipOutputDir)RazorLanguageServer-%(LanguageServiceBinaryDir.Filename)-$(PackageVersion).zip</ZipFile>
```

Comment:
Unless other code may in the future add `@(LanguageServiceBinary)` items, suggest adding the metadata when creating `@(LanguageServiceBinaryDir)` items just above i.e. just use one item group. `%(Identity)` and `%(Filename)` should be available immediately.